doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -214,17 +214,17 @@ To prepare the nRF54H20 DK for first use, you must manually program the values o
 
 #. Move the BICR HEX file to a folder of your choice, then program the BICR by running nRF Util from that folder using the following command::
```

Comment:
I do not think that this step is required. This command can be executed from ``Downloads`` directory.
Additionally - Can we program BICR after switching to ROT LCS?

Reply:
Yes, BICR can be programmed at LCS ROT.